src/app/components/MediaLoader/configs/index.ts

```diff
@@ -1,14 +1,24 @@
-import { PageTypes } from '#app/models/types/global';
-import livePage from './livePage';
-import article from './article';
+import filterForBlockType from '#app/lib/utilities/blockHandlers';
+import clipMedia from './clipMedia';
+import aresMedia from './aresMedia';
+import { AresMediaBlock, ClipMediaBlock, MediaBlock } from '../types';
 
-export default (pageType: PageTypes) => {
-  switch (pageType) {
-    case 'live':
-      return livePage;
-    case 'article':
-    case 'mediaArticle':
-      return article;
+export default (blocks: MediaBlock[]) => {
+  const aresMediaBlock: AresMediaBlock = filterForBlockType(
+    blocks,
+    'aresMedia',
+  );
+
+  const clipMediaBlock: ClipMediaBlock = filterForBlockType(
+    blocks,
+    'clipMedia',
+  );
+
+  switch (true) {
+    case !!aresMediaBlock:
```

Comment:
Is `!!` necessary for each case in this block? Would the switch/case not evaluate it the same without having to explicitly coerce to Boolean?

Reply:
For TypeScript it is necessary as the each `case` is evaluated against whats being switched, which is a boolean `true` in this case.

Reply:
Awesome, thanks for explaining!